crates/actor/src/compat/wasmcloud/bus/host.rs

```diff
@@ -44,7 +44,7 @@ pub fn call(
             // call succeeded
             let mut buf = vec![0; unsafe { __host_response_len() }];
             unsafe { __host_response(buf.as_mut_ptr()) };
-            Ok(buf)
+            Ok(Some(buf))
```

Comment:
When will we return an `Ok(None)`?

Reply:
Here (module using wasmbus) - never, but a component (using wasmCloud WIT) might